internal/cli/atlas/commonerrors/errors.go

```diff
@@ -34,6 +35,9 @@ func Check(err error) error {
 		if apiError.GetErrorCode() == "TENANT_CLUSTER_UPDATE_UNSUPPORTED" {
 			return errClusterUnsupported
 		}
+		if apiError.GetErrorCode() == "GLOBAL_USER_OUTSIDE_SUBNET" {
+			return errOutsideVPN
+		}
```

Comment:
we should show this error message only if the user is a MongoDB employee. We can get the user email address from the auth token

Reply:
I'm doing more testing to see if anyone would get this error

Reply:
@andreaangiolillo I can't find a way to reproduce the problem on an email that is not `@mongodb.com` have you found a way to do that?

Reply:
looking at the backend we only throw this exception is for users with GLOBAL roles (which are only granted to employees anyway)